src/app/components/label-selector/label-selector.component.ts

```diff
@@ -168,40 +168,40 @@ export class LabelSelectorComponent implements OnInit, OnChanges {
       },
       type: 'labels'
     };
-    this.labelService.createLabel(labelPayload)
-      .subscribe((data: LabelModel) => {
-        this.createDisabled = false;
-        this.newSelectedColor = this.colors[Math.floor(Math.random()*this.colors.length)];
-        this.allLabels = [cloneDeep(data), ...this.allLabels];
-        const newLabel = {
-          id: data.id,
-          color: data.attributes['background-color'],
-          border: data.attributes['border-color'],
-          name: data.attributes.name,
-          selected: false
-        };
-
-        // Emit new label
-        // TODO: Should be replaced by ngrx/store
-        this.eventService.labelAdd.next(data);
-
-        this.backup = [cloneDeep(newLabel), ...this.backup];
-        if (this.searchValue === '' ||
-            (this.searchValue !== '' &&
-              name.indexOf(this.searchValue) > - 1
-            )
-          ) {
-            this.labels = [cloneDeep(newLabel), ...this.labels];
-          }
-        this.labelnameInput.nativeElement.value = '';
-        this.labelnameInput.nativeElement.focus();
-      },
-      (err) => {
-        console.log(err);
-        this.labelnameInput.nativeElement.value = '';
-        this.createDisabled = true;
-      }
-    );
+    // this.labelService.createLabel(labelPayload)
+    //   .subscribe((data: LabelModel) => {
```

Comment:
As a best practice can we not commit commented code  -  https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f